backend/src/middleware/notifications/notificationsMiddleware.js

```diff
@@ -105,7 +108,7 @@ const handleCreateComment = async (resolve, root, args, context, resolveInfo) =>
       return record.get('user')
     })
     if (context.user.id !== postAuthor.id) {
-      await notifyUsers('Comment', comment.id, [postAuthor.id], 'comment_on_post', context)
+      await notifyUsers('Comment', comment.id, [postAuthor.id], 'commented_on_post', context)
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-30T12:00:55Z" title="Friday, August 30th 2019, 2:00:55 pm +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -105,7 +108,7 @@ const handleCreateComment = async (resolve, root, args, context, resolveInfo) =>
       return record.get('user')
     })
     if (context.user.id !== postAuthor.id) {
-      await notifyUsers('Comment', comment.id, [postAuthor.id], 'comment_on_post', context)
+      await notifyUsers('Comment', comment.id, [postAuthor.id], 'commented_on_post', context)
```


Did you make an attempt to refactor this @roschaefer? Are we still planning on having a pairing session to have a look at it together?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-30T13:50:02Z" title="Friday, August 30th 2019, 3:50:02 pm +02:00">Aug 30, 2019</time>_

---


It's alright. I did not know that you also save the `reason` to the notification and then make use of this attribute in the frontend. I don't want to refactor it.